src/graphql/schema.gql

```diff
@@ -384,6 +384,7 @@ type Space {
   template: String
   verified: Boolean
   flagged: Boolean
+  hibernated: Boolean
```

Comment:
The issue is with this boolean approach is that we will have to duplicate the hibernation checking logic on the frontend to be able to determine the reason for hibernation and show the right information to the user.

Also what if the users just wanted to change the network from testnet to mainnet in order to de-hibernate their space? This would require manual action from us again?

@ChaituVR @wa0x6e

Reply:
To be honest I liked the https://github.com/wa0x6e/morpheus approach better

Reply:
In care we would go with the PR we could show a generic message that the space has been hibernated and that they need to open a ticket on Discord in order to get it resolved

Reply:
> The issue is with this boolean approach is that we will have to duplicate the hibernation checking logic on the frontend to be able to determine the reason for hibernation and show the right information to the user.
> 
> Also what if the users just wanted to change the network from testnet to mainnet in order to de-hibernate their space? This would require manual action from us again?
> 
> @ChaituVR @wa0x6e

Switching between mainnet and testnet will not be possible soon

Reply:
> Switching between mainnet and testnet will not be possible soon

Right, because sequencer will just reject. We should disable all settings on UI too then

Reply:
> We could show a link to the docs, listing reasons. And there, we can add note to go to discord

@zzuziak could you come up with a flow docs > discord > support flow for de-hibernation 

I will add a message with link to docs on UI

Reply:
@wa0x6e one more issue is though that we can not change space settings. So if a space has a testnet and they request to de-hibernate a space on discord what do we do?

Reply:
> @wa0x6e one more issue is though that we can not change space settings. So if a space has a testnet and they request to de-hibernate a space on discord what do we do?

They should ensure their space conform to the requirement before asking for reactivation

Reply:
As you said `Switching between mainnet and testnet will not be possible soon`

Reply:
We should forbid only proposal creation for hibernated spaces.

Also, people should be able to switch between mainnet and testnet only if they’re on the wrong realm

Reply:
But that's what I said, and you said it will not be possible

Reply:
Ok, so the new workflow from this new hibernation implementation will be:

- Spaces will be hibernated from the backend
- Hibernated space will not be able to create new proposals anymore
- There's 2 main reasons for hibernation: inactivity, and misconfiguration
- For misconfigured spaces, user will have to fix the space settings, then reapply for reactivation from discord

Misconfigutation is often wrong network, or missing validation.

For wrong network (livenet/mainnet), it's always due to users being on testnet, while being on snapshot.org, where testnet support will be dropped. 
In the space network dropdown, all testnet will be removed, beside the one the space is on (if on testnet), once he selected another option, and save, testnet will disappear from dropdown.

Reply:
How do we know if a space if inactive of misconfigured, to know if we should reactivate ? We can't without a service that tells you that. DB column is a simple boolean

Reply:
Agree to all, except that the dropdown input will be empty if the network is from the wrong env

> In the space network dropdown, all testnet will be removed, beside the one the space is on (if on testnet), once he selected another option, and save, testnet will disappear from dropdown.

Reply:
<img width="337" alt="image" src="https://github.com/snapshot-labs/snapshot-hub/assets/51686767/04c3b389-9318-4984-976a-1b188694cd5c">

Reply:
That also works